### DISCUSSION TITLE
Integrating other Repos into RetroPie-Extra

### DISCUSSION THREAD

**Discussion by Gemba** [Exarkuniv/RetroPie-Extra#230] [Ideas]
As I don't want to offload any issues/bug handing to your repo by the contributed modules, I was thinking about options to integrate other github repos into RetroPie-Extra.

Sure, one can put other repos alongside to RetroPie-Extra as the RetroPie-Setup allows this in the ext/ folder but it might be more convenient to have only one repo to update. Whereas RetroPie-Extra has references to other sub repos.

That way RetroPie-Extra has a defined state of the sub repos and also bugs/questions can be redirected to the respective github repo and do not clutter your issue backlog.

I was leaning towards git subtree (instead of submodule), but then there has to be some logic whenever a update is run from RetroPie-Extra and its sub repos via the RetroPie-Setup dialogs.

**Comment by Exarkuniv** [comment]
may not be a bad thing. i know a people have come to me when there was a issue with something, but it was nothing i could do since it was something that was changed upsteam. 

but there was a few times where it was a issue with my install since the repo that i was going to change how they handled things.
